dashboard/app/models/pd/application/teacher_application.rb

```diff
@@ -383,10 +383,10 @@ def should_send_decision_email?
     def log_status
       self.status_log ||= []
       status_log.push({status: status, at: Time.zone.now})
+    end
 
-      # delete any unsent emails, and queue a new status email if appropriate
-      emails.unsent.destroy_all
```

Comment:
We never send unsent emails now. The only way an email would be unsent if there is an error in sending it.